### PR TITLE
Form/add user

### PR DESCRIPTION
**Pull request by OmaritoAtPoly**
this branch gives the form to add a new user

### CODE REVIEW

**Comment by OmaritoAtPoly** [comment]
imagen

File: package.json [hunk 1]
```diff
@@ -14,8 +14,10 @@
     "@types/bcryptjs": "^2.4.2",
     "@types/jsonwebtoken": "^8.5.0",
     "@types/moment": "^2.13.0",
+    "@types/yup": "^0.29.0",
```
Comment (elyanio) [inline package.json:17]: the typescript dependency should by only in in dev-dependency

File: src/AddUserForm.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        margin: '5rem auto'
+    },
+    title: {
+        display: 'flex',
+        justifyContent: 'center',
+    },
+    input: {
+        width: '100%',
+    },
+    form_group: {
```
Comment (elyanio) [inline src/AddUserForm.tsx:96]: use camelCase

File: src/App.tsx [hunk 3]
```diff
@@ -1,15 +1,10 @@
...
 import React from "react";
+import { ClientContextProvider } from "react-fetching-library";
 import { BrowserRouter } from "react-router-dom";
+import { Client } from './helpers/Client';
 import { Router } from "./routes/route";
 import { Theme } from "./theme";
-import { createClient, ClientContextProvider } from "react-fetching-library";
-import {Client} from './helpers/Client';
-
-// const client = Client({
```
Comment (elyanio) [inline src/App.tsx:9]: why do you delete that?

Comment (OmaritoAtPoly) [reply src/App.tsx:9]: because I used another type to create the client to use the hook, not using an empty array.

Comment (elyanio) [reply src/App.tsx:9]: please, help me, why do you use `requestHostInterceptor(HOST!)`?, I don't understand

Comment (OmaritoAtPoly) [reply src/App.tsx:9]: requestHostInterceptor(HOST!) it's a function which need  the contained URL of the API, beside the last part of the path, "/users" to get users + the method "get/post/put" etc

**Pull request merged**